my-app/src/AddBox/AddBox.jsx

```diff
@@ -103,38 +90,44 @@ export default class AddBox extends Component {
   };
 
   handleTagSelection = (tag) => {
-    this.props.addTagToPerson(tag.label);
+    this.props.addTagToPerson(tag.label, this.state.publicity);
   }
 
-  handlePersonSelection = (person) => {
-     this.props.setPerson(person);
+   handleLabelSelection = (label) => {
```

Comment:
this function and the above are the same ---> 👀  Did I read correctly? I dunno. It is kind of late.

Reply:
They're slightly different - `handleTagSelection` needs to pull the label out of the tag and feed it to `addTagToPerson` (`tag.label`), while `handleLabelSelection` feeds the label in directly